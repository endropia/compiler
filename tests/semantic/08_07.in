var
	arr: array[0..10] of integer;
begin
	writeln(arr);
end.