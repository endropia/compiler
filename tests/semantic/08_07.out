It can not be used in io procedures